frontend/src/scenes/data-warehouse/new/dataWarehouseTableLogic.tsx

```diff
@@ -125,4 +133,9 @@ export const dataWarehouseTableLogic = kea<dataWarehouseTableLogicType>([
             },
         },
     })),
+    events(({ actions }) => ({
+        propsChanged: () => {
+            actions.loadTable()
```

Comment:
How often do props change here? Wanna make sure we wont be making a bunch of requests on loop here